tests/command.dev.test.js

```diff
@@ -333,7 +385,7 @@ testMatrix.forEach(({ args }) => {
 
       await builder.buildAsync()
 
-      await withDevServer({ cwd: builder.directory, env: { CONTEXT: 'production' }, args }, async (server) => {
+      await withDevServer({ cwd: builder.directory, args }, async (server) => {
```

Comment:
I think we should allow overriding it via process level env, but can go either way